src/data/vault.json

```diff
@@ -161,6 +161,11 @@
 			"iconName": "api",
 			"name": "APIs",
 			"path": "api-docs"
+		},
```

Comment:
Can we put tutorials directly after Vault so it's second in the list? I know it was at the end before, but it makes more sense to get l group it together with docs instead of the reference tabs